### DISCUSSION TITLE
Interstitial Bound Fraction

### DISCUSSION THREAD

**Discussion by pburkat** [Open-Systems-Pharmacology/Forum#1193] [Questions & Problems]
Hello, how can the protein bound fraction of a compound in the interstitial space be derived? I see the total interstitial space concentration, and the unbound fraction is the product of the the water:interstitial partition coefficient and total interstitial concentration. Is there a protein:interstitial partition coefficient for this calculation, or is there another method? Thanks.

**Comment by StephanSchaller** [comment]
see my other comment. The interstitial/water PC value is what you need to look for
